src/Akka.Remote.Hosting/RemoteOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="AkkaRemoteOptions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Akka.Configuration;
+
+namespace Akka.Remote.Hosting
+{
+    public sealed class RemoteOptions
+    {
+        public string? HostName { get; set; }
+        public int? Port { get; set; }
+        public string? PublicHostName { get; set; }
+        public int? PublicPort { get; set; }
```

Comment:
I would add an XML-DOC comment saying that this param is optional and virtually no one needs this - only in instances where you need to map a public port to a different private port is this necessary.